__tests__/app.test.js

```diff
@@ -32,8 +32,13 @@ describe("1.getCategories", () => {
             .then((res) => {
                 let categories = res.body.categories;
                 expect(Array.isArray(categories)).toBe(true);
+                categories.forEach(cat => {
```

Comment:
We should test the length of the array, as this is done with test data we should know what were getting back.
This also protects the forEach as that would pass if an empty array is passed.